integration/tests/conftest.py

```diff
@@ -60,6 +60,7 @@ def record_test_metric(request):
                 'host': socket.gethostname(),
                 'user': getpass.getuser(),
                 'run-id': os.getenv('TEST_METRICS_RUN_ID', None),
```

Comment:
getenv each time or save the values upfront?

Reply:
I don't feel strongly either way. I think it's fine to leave them as is, but I'm happy to change if you prefer doing it upfront.